### PR TITLE
You can now manually link stasis beds to operating computers

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

This allows you to directly link a stasis bed to an operating computer using a multitool, no more reconstructing stasis beds to try to get it to link to the right computer.

Also, surgery tables and stasis beds will now only try to link to unused operating computers, rather than cannibalizing an operating computer already linked to something.

## Why It's Good For The Game

Because getting the right stasis bed to link with the right operating computer, when building them near each other, can be frustrating.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

23-03-31-1680281036-dreamseeker

note: hitting it with the multitool should be fixed now

</details>

## Changelog
:cl:
add: You can now link stasis beds to operating computers with a multitool!
tweak: Surgery tables and stasis beds will now only try to link to unused operating computers.
/:cl:

### CODE REVIEW

**Comment by Pirill** [comment]
The heroic QoL this station needed!

**Comment by Fronsis** [comment]
A little nit-picky but can you make it so the multitool doesn't actually *hit* the machine? From the description on the picture seems like it took damage after a hit, obviously i doubt you'll ever end up breaking it by just multitooling multiple times for some strange coindicence but i think that other types of multitool linking doesn't actually harm the machine

**Comment by Absolucy** [comment]
@Fronsis I fixed that between taking the video and making this PR.

File: code/game/machinery/computer/Operating.dm [hunk 1]
```diff
@@ -162,5 +160,24 @@
 			. = TRUE
 	. = TRUE
 
+/obj/machinery/computer/operating/multitool_act(mob/living/user, obj/item/I)
```
Comment (PowerfulBacon) [inline code/game/machinery/computer/Operating.dm:163]: This has no range check so you can link them from infinitely far away

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
The code in isolation is fine, however multitool's buffer will result in hard dels and needs refactoring. Increased uses of  this will increase the likelyhood of these hard dels happening

File: code/game/machinery/computer/Operating.dm (hunk 1 above)
Comment (Absolucy) [reply code/game/machinery/computer/Operating.dm:163]: Oh, huh, I was thinking tool_act would handle adjacency checks. Alright.

Comment (Absolucy) [reply code/game/machinery/computer/Operating.dm:163]: Actually, no other `multitool_act` implementations seem to handle range checks themselves?

Comment (PowerfulBacon) [reply code/game/machinery/computer/Operating.dm:163]: There is no range limit to linking to the computer I mean, not with using the multitool. It should have a range of like 2 or 3 max so you can't link it to the surgery computer and then just unlock all the surgeries without actually moving the computer.

Comment (Absolucy) [reply code/game/machinery/computer/Operating.dm:163]: Ohh. Fixed.

**Review by itsmeow — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**